projects/packages/forms/src/contact-form/class-contact-form.php

```diff
@@ -659,17 +658,30 @@ public static function get_compiled_form_for_email( $feedback_id, $form ) {
 
 					$field_label = $field->get_attribute( 'label' ) ? $field->get_attribute( 'label' ) . ':' : '';
 
-					$compiled_form[ $field_index ] = sprintf(
-						'<p><strong>%1$s</strong><br /><span>%2$s</span></p>',
+					$compiled_form[ $field_index ] = array(
 						wp_kses( $field_label, array() ),
-						self::escape_and_sanitize_field_value( $extra_fields[ $extra_field_keys[ $i ] ] )
+						self::escape_and_sanitize_field_value( $extra_fields[ $extra_field_keys[ $i ] ] ),
 					);
 
 					++$i;
 				}
 			}
 		}
 
+		$updated_compiled_form = apply_filters( 'jetpack_forms_response_email', $compiled_form, $feedback_id, $form );
```

Comment:
It would be nice to have docblocks for each one of the new filters, to exactly explain what they can be used for. That documentation is then used to automatically generate [the Jetpack Codex](https://developer.jetpack.com/) on each release. Given [the amount of the filters we already have for the contact form feature](https://developer.jetpack.com/module/contact-form/), it's going to be important that we document the exact use-case for each filter so no one feels overwhelmed or confused as to what filter to use.

You can check the existing filters in these files to get an idea of the expected format of those docblocks. They should all include:
- A title
- `@module contact-form`
- `@since $$next-version$$` (that placeholder is then automatically replaced by the right version number on release)
- a `@param` entry for each parameter.

When documenting filters and the filter is used in multiple files, you can document usage in one place, and only refer to the full documentation in the other. Here is an example:
https://github.com/Automattic/jetpack/blob/0e4a3203030c28c1cb5b6a16b62ccd6a94a1bdf5/projects/plugins/jetpack/modules/contact-form/grunion-contact-form.php#L3741

Reply:
Thank you for the great explanation. I'll add those.